rai-sdk/src/main/java/com/relationalai/Client.java

```diff
@@ -81,9 +89,10 @@ public Client(Config cfg) {
         this.httpClient = HttpClient.newBuilder().build();
         this.credentials = cfg.credentials;
         this.setAccessTokenHandler(new DefaultAccessTokenHandler());
+        this.logger = logger == null ? LoggerFactory.getLogger("rai") : logger;
```

Comment:
@torkins can you check the logger creation at this level

Reply:
move this defaulting behavior into the Logger-less ctor overload